portable_mds/tests/test_mds.py

```diff
@@ -13,28 +12,12 @@
     test_custom_warn,
     test_double_run_stop,
     test_event_descriptor_insertion,
-    # test_fail_runstart,
+    test_fail_runstart,
     test_find_run_start,
     test_find_run_stop,
     test_insert_run_start,
-    # test_no_evdesc,
-    # test_pickle,
+    test_no_evdesc,
+    test_exceptions_are_mds_attributes,
+    test_pickle,
```

Comment:
This is the offending test, needs to be left commented out (or be paraterized in mds)